### PR TITLE
Add udf to deanonymize event tuples

### PR DESCRIPTION
**Pull request by sunahsuh**
Whoo, go @acmiyaguchi for getting event tuples working in BQ! This UDF renames the anonymous struct fields in event tuples into meaningful names

### CODE REVIEW

File: udf/deanonymize_event.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+CREATE TEMP FUNCTION
+  udf_deanonymize_event(tuple STRUCT<
+    f0_ INT64,
+    f1_ STRING,
+    f2_ STRING,
+    f3_ STRING,
+    f4_ STRING,
+    f5_ ARRAY<STRUCT<key STRING, value STRING>>>) AS (
+    (SELECT
+      AS STRUCT tuple.f0_ AS `timestamp`,
```
Comment (jklukas) [inline udf/deanonymize_event.sql]: Elsewhere, we have prefixed all of these field names with `event_` which has some benefits in avoiding reserved words like `timestamp` and making the purpose of the field more obvious when unnesting. Should we do so here?

Also, we should add calls to this function into the views for `main` and other pings, but that can be a separate PR.

Comment (sunahsuh) [reply udf/deanonymize_event.sql]: I left these as unprefixed since these fields are still encapsulated within a struct, and it seemed a bit redundant in that context. I can see the argument for the adding the prefix, though for the flattened events dataset my plan was to manually add the prefix in the new query

Comment (sunahsuh) [reply udf/deanonymize_event.sql]: Although tbf the javascript udf _that I wrote a few weeks ago_ that parsed out events had the prefix and that output was also a struct, so I guess I have to go discuss this with my past self

Comment (relud) [inline udf/deanonymize_event.sql]: this should use `STRUCT(` instead of `(SELECT AS STRUCT`, because BigQuery can only handle a limited number of expression subqueries.
